logic/src/commonTest/kotlin/com/wire/kalium/logic/data/message/receipt/ReceiptRepositoryTest.kt

```diff
@@ -72,11 +109,19 @@ class ReceiptRepositoryTest {
             )
         )
         private val TEST_OTHER_USER_ENTITY = TestUser.ENTITY.copy(
+            name = "First User",
```

Comment:
If you wanna be really clear about the names here:

```suggestion
            name = "AAA - First User alphabetically",
```